### DISCUSSION TITLE
Add fleets to a system

### DISCUSSION THREAD

**Discussion by adegans** [endless-sky/endless-sky#7853] [Balance, Mechanics, and Interactions]
Say I want to add 2 fleets to Sol.

Do I really have to redefine the whole system in my plugins map.txt?? 😬
Seems very fragile like this with A LOT of potential to break other plugins.

Even triggering an event with the new fleet definitions overruled all the original stuff...

**Comment by Zitchas** [comment]
You should be able to add a fleet using the `add fleet` keyword

https://github.com/endless-sky/endless-sky/wiki/CreatingEvents

**Reply by adegans** [reply]
I rather add them in a map.txt because the fleets should always be able to spawn.
Doing that seems to clear out the system (planets and everything) except for my 2 fleets.

Plugins map.txt:
```
system "Sol"
	# Added for SpaceCrate
	fleet "spacecrate_merchantfleet" 1000
	fleet "spacecrate_merchantfleet" 1200
```
Gives me a empty system with just 2 fleets in it. So I ended up copying the entire thing from the games map.txt and adding my 2 fleets.

Alternatively I tried the following -  Added a dummy mission that triggered the event on complete.

My plugins events.txt
```
event "Some_thing_1"
	system "Sol"
		# Added for SpaceCrate
		fleet "spacecrate_merchantfleet" 1000
		fleet "spacecrate_merchantfleet" 1200
```
Replaces all fleets in Sol with these 2.

**Reply by quyykk** [reply]
Do this:

```
system "Sol"
	add fleet "spacecrate_merchantfleet" 1000
	add fleet "spacecrate_merchantfleet" 1200
```

**Reply by adegans** [reply]
Aha! That's way better.
Perhaps put in the manual that that also works for things other than events.

Thanks for the tip.
